### PR TITLE
Minor fixes for clarity.

### PR DESCRIPTION
**Pull request by robinpercy**
* Confirming network policies are enabled using gcloud instead of UI
* Creating namespace in manifest instead of as distinct step
* Remove blocked client from second validation section since it was
redundant

### CODE REVIEW

File: manifests/network-policy-namespaced.yaml [hunk 1]
```diff
@@ -12,11 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Defines a new namespace used to demonstrate
+kind: Namespace
```
Comment (chrislovecnm) [inline manifests/network-policy-namespaced.yaml]: Please add URL to the docs here

Comment (robinpercy) [reply manifests/network-policy-namespaced.yaml]: Done

**Comment by chrislovecnm** [comment]
Well CI is not happy 😂 

Is it bad that I am happy about this??

**Comment by chrislovecnm** [comment]
test this please

**Comment by chrislovecnm** [comment]
```
[gke-network-policy-demo] Running shell script
+ make validate
/home/jenkins/workspace/gke-network-policy-demo/validate.sh
user is: jenkins
Makefile:28: recipe for target 'validate' failed
make: *** [validate] Error 1
```
CI failed

**Comment by techgnosis** [comment]
Yeah like Chris pointed out the validate make target does not work. Now that Jenkins is working that part has to be fixed.

**Comment by geojaz** [comment]
@robinpercy do you have any bandwidth to get your tests on this working? I'd like to get this one merged

**Pull request merged**